connectdialog.ui

```diff
@@ -97,6 +100,116 @@
     <widget class="QComboBox" name="flowControlList"/>
    </item>
    <item row="3" column="0" colspan="4">
+    <widget class="QGroupBox" name="dumpFile">
+     <property name="sizePolicy">
+      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
+       <horstretch>0</horstretch>
+       <verstretch>0</verstretch>
+      </sizepolicy>
+     </property>
+     <property name="minimumSize">
+      <size>
+       <width>80</width>
+       <height>120</height>
+      </size>
+     </property>
+     <property name="styleSheet">
+      <string notr="true">QGroupBox {  border: 1px solid rgb(130, 64, 166);}</string>
```

Comment:
Si questa riga di css l'avevo mesa per provare a vedere il frame del QGroupBox. Poi mi sono concentrato sul layout dei elementi e non sono arrivato al punto dove volevo arrivare. Quindi la riga l'ho lasciata perché volevo vedere con te come si aggiunge dal inizio e 'in a clean way' un groupbox e i suoi elementi...